src/HW_models/NRF_RTC.c

```diff
@@ -125,15 +124,52 @@ static void time_of_1_counter_wrap(int rtc, bs_time_t *us, uint16_t *frac) {
   *us = counter_to_time((uint64_t)RTC_COUNTER_MASK + 1, rtc, frac);
 }
 
+/**
+ * Convert a counter value to absolute time.
+ * @note To obtain a time delta, use @ref counter_to_time instead
+ */
+static bs_time_t counter_to_abs_time(int rtc, uint32_t rtc_counter) {
+  uint16_t frac;
+  bs_time_t bs_time = RTC_counter_startT[rtc]
```

Comment:
A very quick comment after a very quick look: bs_ prefixes are used in the babblesim global symbols and types to avoid name classes. It does not make much sense (and does not help with readability) to add them to local variables.

Reply:
- [ ] Do not use the `bs_` prefix for local variables

Reply:
Fixed